.github/workflows/static-binary.yaml

```
@@ -71,6 +71,9 @@ jobs:
         STATIC_BINARY_FOLDER: vast-static-builds
       run: |
         gsutil -m cp "${{ env.BUILD_DIR }}/${{ steps.create_paths.outputs.artifact_name }}" "gs://${{ env.PUBLIC_GCS_BUCKET }}/${{ env.STATIC_BINARY_FOLDER }}"
+        RELEASE_MONTH=$(echo "${{ steps.create_paths.outputs.artifact_name }}" | cut -d"-" -f2)
+        gsutil -m cp "${{ env.BUILD_DIR }}/${{ steps.create_paths.outputs.artifact_name }}" "gs://${{ env.PUBLIC_GCS_BUCKET }}/${{ env.STATIC_BINARY_FOLDER }}/vast-${RELEASE_MONTH}-static-latest.tar.gz"
```

`-m` seems to only affect the case where multiple files are transmitted.

Would it be possible to use the previous copy for the source like so?
```suggestion
        gsutil -m cp "gs://${{ env.PUBLIC_GCS_BUCKET }}/${{ env.STATIC_BINARY_FOLDER }}/${{ steps.create_paths.outputs.artifact_name }}" "gs://${{ env.PUBLIC_GCS_BUCKET }}/${{ env.STATIC_BINARY_FOLDER }}/vast-${RELEASE_MONTH}-static-latest.tar.gz"
```